src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -378,7 +378,9 @@ public async Task<IActionResult> DeletePipelineFromChannel(int channelId, int pi
 
             if (targetChannel != null)
             {
-                flowGraph.PruneGraph(node => IsInterestingNode(targetChannel, node), edge => IsInterestingEdge(edge, includeDisabledSubscriptions, frequencies));
+                flowGraph.PruneGraph(
+                    node => flowGraph.IsInterestingNode(targetChannel.Name, node), 
```

Comment:
I like this implementation. I was thinking you were going to just end up removing the possibility of custom pruning, but I like this much better. I think those methods can be made static though.

Reply:
Made them static.